org.freedesktop.LinuxAudio.Plugins.Cardinal.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<component type="addon">
+  <id>org.freedesktop.LinuxAudio.Plugins.Cardinal</id>
+  <extends>org.freedesktop.LinuxAudio.BaseExtension</extends>
+  <name>Cardinal</name>
+  <summary>Cardinal is a free and open-source virtual modular synthesizer plugin</summary>
+  <project_license>GPL-3.0</project_license>
```

Comment:
shouldnt this use SPDX identifier?

Reply:
I was following the example of other flathub repos. Should it be "GPL-3.0-only" or "GPL-3.0-or-later"?

Reply:
GPL-3.0-or-later